app/src/androidTest/java/com/sdp/movemeet/Login/LoginActivityTest.java

```diff
@@ -103,5 +103,5 @@ public void Login_TrueNonEmpty(){
         // Start the screen of your activity.
         onView(withId(R.id.nav_view)).perform(NavigationViewActions.navigateTo(R.id.nav_logout));
```

Comment:
So some tests of the LoginActivityTest don't pass and you commented them out? I guess JL made most of those tests work in his last pull request.

Reply:
Those tests were already commented in the main, we'll have to take a look on it

Reply:
Effectively, it didn't pass, but we keep it for the future!